Ubuntu-clang-latest/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM cgal/testsuite-docker:ubuntu
+
+MAINTAINER Maxime Gimeno
+
+#install the necessary packages
+RUN apt-get update \
+ && apt-get install -y python \
+ && apt-get install -y subversion 
+
+#download llvm and clang from trunk
+RUN cd /tmp \
+ && svn co http://llvm.org/svn/llvm-project/llvm/trunk llvm \
+ && cd ./llvm/tools \
+ && svn co http://llvm.org/svn/llvm-project/cfe/trunk clang 
+
+#build llvm and clang
+RUN cd  /tmp/ \
+ && mkdir build \
+ && cd ./build \
+ && cmake -DCMAKE_CXX_COMPILER:FILEPATH=/usr/bin/g++ ../llvm \
+ && make -j"$(nproc)"
+
+#Clean-up and set some ENV variables
+RUN cd /tmp \
+ && rm -rf ./llvm
+ENV PATH="/tmp/build/bin:${PATH}"
```

Comment:
A Docker "layer" is created for each command in the Dockerfile. It means that at the `RUN` that build, the layer is very big, even if at the next `RUN` a lot of files are removed.

It would be best to:
```dockerfile
RUN  /tmp/ \
&& mkdir build \
&& cd ./build \
&& cmake -DCMAKE_CXX_COMPILER:FILEPATH=/usr/bin/g++ ../llvm \
&& make -j"$(nproc)" \
&& make -j"$(nproc)" install 
&& cd /tmp && rm -rf build llvm
```
And that would be even better to merge that `RUN` with the one that does the `svn co`. That would reduce the size the image (that is the sum of sizes of layers).